service-front/module/Application/tests/Model/Service/Lpa/CommunicationTest.php

```diff
@@ -45,15 +46,90 @@ public function setUp(): void
         $this->service->setUserDetailsSession($userDetailSession);
     }
 
-    public function testSendRegistrationCompleteEmailWithoutPayment(): void
+    public function testSendRegistrationCompleteEmailWithoutPaymentButWithPersonToNotify(): void
```

Comment:
Did you check coverage for the unit tests? There are a lot of conditionals? Might be worth making sure you caught all the branches.

Reply:
I believe that all the cases where a payment is made, are covered by the unit tests.  For no payment being made, I've extended this to have seperate tests for Universal Credit, Receives Benefits, Awarded Damages,  in addition to the Low Income one. These extra tests are identical to the Low Income one except for the single param set on the payment .  I hadn't originally included them as expected behaviour was identical. However, adding them now protects us if we want  varying behaviour in future